src/main/resources/db/changelog/changes/01-create-books-table.yaml

```diff
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books-table
+      author: haiovyi
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
primary key can't be null by default, we can delete this constrain

Reply:
and is it actually ok to leave that there? )

Reply:
Better to remove, It could be confusing

Reply:
fixed